### DISCUSSION TITLE
作者你好，用于中英文训练的数据集不知是否方便告知。

### DISCUSSION THREAD

**Discussion by z20150214** [OleehyO/TexTeller#55] [Q&A]
对于公式中穿插中文的识别，在使用对应的数据集对比中文单独训练，识别时会有很大的影响吗

**Comment by OleehyO** [comment]
加入中文后对正常公式的识别似乎没有很大影响，这个之后可能要做个实验验证一下。

数据集的话最后肯定会开源的（等论文发表后，预计在8月份之后）。
